centreon/packages/ui/src/components/ExpandableContainer/ExpandableContainer.tsx

```diff
@@ -0,0 +1,63 @@
+import { OpenInFull } from '@mui/icons-material';
+import { CSSProperties, useState } from 'react';
+import { Modal } from '../Modal';
+import { useStyles } from './expandableContainer.styles';
+import { Parameters } from './models';
+import { labelExpand, labelReduce } from './translatedLabels';
+
+interface Props {
+  children: (params: Parameters) => JSX.Element;
+  style?: CSSProperties;
+}
+
+const ExpandableContainer = ({ children, style }: Props) => {
+  const { classes } = useStyles();
+
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  const toggleExpand = () => {
```

Comment:
```suggestion
  const toggleExpand = (): void => {
```
Don't forget to add some memoization on functions and variables